README.md

```diff
@@ -43,6 +43,17 @@ To the following:
 
 You can also set this specifically for the project directory. You may need to restart the server afterward.
 
+## Run with Docker
+
+An alternative to running pvpoke locally is using a docker container. That way you can ignore all the steps in the Installation-Part. Instead you have these steps:
+
+1. Clone this repository.
+2. If you don't have Docker installed, install it [here](https://runnable.com/docker/getting-started/).
+3. Go into this directory with the Dockerfile in it.
+4. Run this command:
+```docker build -t pvpoke . & docker run -p 80:80 pvpoke```
```

Comment:
This caused a bit of trouble for me if it’s a subsequent build - it doesn’t wait for the build to finish and immediately attempts the `run`, using the previous image. Maybe `&&`?

Reply:
@wasabigeek Good point, I've changed the command in the README.md according to your proposal. & -> &&

About mounting the source code: To be honest I wasn't entirely working with docker alone. The company I'm working for, covexo GmbH develops a tool called devspace. This tool works on top of docker and kubernetes and its main purpose is to help developing apps in docker containers.
The idea: You run "devspace dev", which starts a docker container with your code in it. (This command can even generate Dockerfiles. Even the Dockerfile I added in this PR is mostly generated). Devspace dev also adds a sync path between the local and container directory. This means, whenever code is changed, the change is synced live into the container without the need of running a new command.

Why am I bringing this up? You suggested to mount the code so the images don't have to be rebuild, but I think with devspace it is even easier.

Btw I'm not sure whether this is unwanted advertising. I'm clearly biased. Therefore I hesitated t bring it up. But now you spoke about an issue,  devspace is designed to do better so I had to bring it up. 

https://github.com/devspace-cloud/devspace

Reply:
hello, I am trying to use the docker steps in readme but cannot get it work. 

I am getting this issue:
`Could not reliably determine the server's fully qualified domain name, using 172.19.0.2. Set the 'ServerName' directive globally to suppress this message`

Fairly new to Docker overall.